### DISCUSSION TITLE
Table: skeleton rows should be the same amount of itemsPerPage

### DISCUSSION THREAD

**Discussion by EmanueleQuerzola** [wonderflow-bv/wanda#158] [Ideas]
### Feature description

To keep the loading fase more consistent we should align the number of rows' skeleton to the itemsPerPage number of the table.

### Code of Conduct

- [X] I've searched across open and closed issues for similiar requests

**Comment by equinusocio** [comment]
`itemsPerPage` is `undefined` if pagination is not enabled. Where there is no pagination, this would lead to a possibly infinite list of useless skeletons.

If `itemsPerPage` is set to `50` is nonsense to show 50 skeletons since most of them are outside the viewport.

If there is no pagination and so `itemsPerPage` is undefined, the skeleton count should match the number of the rows, which is `unknown`.


> **Warning**
> There are many issues behind this change, and it will bring a lot of complexity I don't think we need for a loading indicator.

**Comment by equinusocio** [comment]
One possible implementation could be using `pageSize` checking if exists and is `>= 1 and <= X`.

Supposing `X` is `15` (the maximum number of skeletons we want to show), if `pageSize` is between 1 and 15, we show the exact number of skeletons, if greater than 15 or undefined (no pagination) we show 15 skeletons

**Comment by equinusocio** [comment]
Closing cause lack of interest.
